packages/Interface/src/DTK_C_API.cpp

```diff
@@ -315,6 +323,12 @@ const char *DTK_git_commit_hash()
 
 DTK_UserApplicationHandle DTK_create( DTK_ExecutionSpace space )
 {
+    if ( !DTK_is_initialized() )
+    {
+        errno = DTK_UNINITIALIZED;
```

Comment:
No, I think `errno` is global. We do not have `using namespace std`, but it compiles fine.